GSoC/2015/Julia-Quantum-:-Framework-for-Solvers.md

```diff
@@ -0,0 +1,185 @@
+# Julia Quantum : Framework for Solvers
+
+### Contact Info
+
+Name: Amit   
+Email: bitsjamadagni@gmail.com   
+Location/Timezone: India, GMT +5:30   
+University: Birla Institute of Technology and Science-Pilani, India     
+
+## Background:
+### Motivation for the project:
+
+I am a pre-final year undergraduate pursuing Masters in Mathematics along with Bachelors in Electronics and Electrical from BITS-Pilani, India. Being a Masters student in Mathematics I have been exposed to courses like Analysis, Algebra, Topology, Differential Geometry among many others. My interest for physics got me started with Quantum Mechanics, and this interest intensified as I started to realize the abstract nature of math being applied to develop axiomatic physical theories. Working with huge software libraries always inspired me to construct one from ground zero. I was in search of a library where I could put my efforts in this direction, and I came across Julia. As I searched further I was excited to see Julia Quantum package which had intersections with my interests and since then have tried to contribute to it.
+
+### Programming Skills:
+
+My journey with open source started with SymPy where I had fixed some bugs in Quantum physics module, my interest in Topological Quantum Computation introduced me to the low dimensional topological based ideas of Links/Knots which further led to a Google Summer of Code 2014 project for Sage Mathematical Software where I was involved in developing Computational Knot theory library. The project was based on python and it mainly dealt with conversions between various representations of Links/Knots, on the top of it there were few invariants implemented to distinguish between various Links/Knots. In addition my interest towards hardware allowed me undertake projects involving Raspberry Pi, the details of which could be found on my GitHub repo (Repo titles : PiSMS, Project). I had an opportunity to contribute to Julia and Julia Quantum in the form of enhancing few features. This project would further enhance my knowledge in various domains ranging from Quantum Mechanics to further understanding of Julia. 
+ 
+* GitHub Profile : https://github.com/amitjamadagni
+* GSoC 2014 : https://www.google-melange.com/gsoc/project/details/google/gsoc2014/amitjamadagni/5676830073815040
+* GSoC 2014 blog : https://knotsknotted.wordpress.com/category/gsoc/
+* Contribution to Julia : Implementation of ror! and rol! (https://github.com/JuliaLang/julia/pull/9822)         
+* Contribution to Julia Quantum : Normalize to QuArray (https://github.com/JuliaQuantum/QuBase.jl/pull/15)           
+
+---
+### Project Synopsis : The main aim of the project is to achieve 
+
+1. Framework to solve SE or ME by
```

Comment:
or perhaps SE is really Schroginer equation.  Would be best to expand these abbreviations, I think.